### ISSUE TITLE
[BUG] InputManager calls non-existent function ZInput/ButtonDef (since Bog Witch update)

### ISSUE THREAD

**Issue by SpikeHimself** [Valheim-Modding/Jotunn#439]
Details:
Jotunn Version: 2.21.1
Jotunn Submodule(if applicable): InputManager
Repeatability(Consistent(100%), Inconsistent(50%), Rare(1%), Unknown(==1)): 100%

**Problem Description:**
My mod XPortal adds custom buttons for gamepad support via Jotunn's InputManager. Further down the line this causes the game to fail loading captions in the menu, and causes it to crash completley when opening and changing the graphics settings in the game.

**Link to your server's and client's LogOutput.log**
Please refer to SpikeHimself/XPortal#105 for further details and log output

**Comment by SpikeHimself** [comment]
Full logs available in the link provided, but here's the short version where I think the root cause is:
```
MissingMethodException: Method not found: ZInput/ButtonDef .ZInput.AddButton(string,GamepadInput,single,single,bool,bool)
Stack trace:
Jotunn.Managers.InputManager+Patches.RegisterCustomInputs (ZInput __instance) (at D:/Projects/Jotunn/JotunnLib/Managers/InputManager.cs:179)
```

**Comment by MSchmoecker** [comment]
Thanks for the report, this will be fixed by #438

**Comment by SpikeHimself** [comment]
You're the real Hero here, thank you as always for being on top of things!

**Comment by MSchmoecker** [comment]
Fixed in Jotunn 2.21.2
